src/constants.js

```diff
@@ -273,6 +273,15 @@ export const TOPIC_PATH = '/subjects/:subjectId/:topicPath(.*)?/:topicId';
 export const SUBJECT_PAGE_PATH = '/subjects/:subjectId';
 export const SUBJECTS = '/subjects';
 export const FILM_PAGE_PATH = '/subjects/subject:20';
+export const PLAIN_LEARNINGPATH_PAGE_PATH = '/learningpaths/:learningpathId';
```

Comment:
Url for en ressurs er lik uavhengig av om det er en artikkel eller en læringssti. Så ha /subjects som første ledd av url i stedet for /learningpaths.